exercises/risk-risiko.cpp

```diff
@@ -27,3 +27,58 @@
   M 3 vs 3 => blue win
   O 2 vs 1 => red win
 */
+
+#include <iostream>
+#include <algorithm>
+#include <cstdlib>
+using namespace std;
+
+
+int attacker_Dices[3];
+int defender_Dices[3];
+  
+void compare(){ 
+    sort(attacker_Dices, attacker_Dices+ 3, greater<int>());
+    sort(defender_Dices, defender_Dices + 3, greater<int>());
+    for(int i=0; i<3; i++){
+        if(defender_Dices[i]>=attacker_Dices[i])
+            {
+                cout<<"N"<<attacker_Dices[i]<<" vs  "<<defender_Dices[i]<< 
+" ==> Blue Win"<<endl;
+            }
+        else  
+            {
+                cout<<"N"<<attacker_Dices[i]<<" vs  "<<defender_Dices[i]<< 
+" ==> Red Win"<<endl;
+            }
+    }
+
+}
+  
+int main(){
+    srand(rand()%556745+1);
+
+
+    cout<<"Red Dices:\n";
+     for(int i=0; i<3; i++){
+        attacker_Dices[i]= rand() % 5+1;
+        cout<<attacker_Dices[i]<<endl;
+   
+    
+ }
+
+ 
+       cout<<"Blue Dices:\n";
+        for(int i=0; i<3; i++){
```

Comment:
Potresti eliminare questo for e utilizzare il precedente per fare le operazioni sul secondo array visto che esse sono indipendenti fra loro, quindi evitare una serie di cicli.